src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -193,7 +197,7 @@ private async Task FillRequestLists(List<Hl7.Fhir.Model.Bundle.EntryComponent> b
                     RouteData = routeContext.RouteData,
                 };
 
-                _requests[entry.Request.Method.Value].Add(routeContext);
+                _requests[entry.Request.Method.Value].Add((routeContext, order++));
```

Comment:
what happen if the entry.Request.Method is not one that's defined in the dictionary? For example, Patch is not defined in Stu3 BundleHandler but it doesn't stop me from specifying Patch in the request as one of the request. Is there validation that happens before this?

Reply:
Oh nevermind, I see that this is HTTPVerb enum which is defined differently for each version.